app/models/user.rb

```diff
@@ -18,17 +16,13 @@ class User < ActiveRecord::Base
 
   scope :with_statistics, Queries::StatisticsQuery
 
-  def self.from_omniauth(auth)
-    email_address = auth.info.email
-    grabbed = EMAIL_FORMAT.match(email_address).try(:[], :email)
-    grabbed = grabbed ? "#{grabbed}%" : email_address
-    user = where("email LIKE :email", email: grabbed).first_or_create do |u|
-      u.name= auth.info.name
-      u.email= auth.info.email
-      u.image = auth.info.image
+  def self.from_andela_auth(user)
+    user = where("email LIKE :email", email: user['email']).first_or_create do |u|
+      u.name= user['name']
+      u.email= user['email']
+      u.image = user['picture']
```

Comment:
This saves the user's image url huh? In the event that someone changes their profile pix, we won't get an update on this field because our query would have found the email and moved on, I guess. Is this what we want?